app/containers/NewMediaCall/NewMediaCall.integration.test.tsx

```
@@ -0,0 +1,412 @@
+			Object.defineProperty(this, 'sessionId', { value: `session-${config.userId}`, writable: false });
+			createdSessions.push(this);
+		})
+}));
+
+// ─── Helpers ──────────────────────────────────────────────────────────────────
+
+function makeIncomingCall(options: {
+	callId?: string;
+	role?: 'caller' | 'callee';
+	hidden?: boolean;
+}): IClientMediaCall {
```

ESLint

[error] Replace `⏎↹callId?:·string;⏎↹role?:·'caller'·|·'callee';⏎↹hidden?:·boolean;⏎` with `·callId?:·string;·role?:·'caller'·|·'callee';·hidden?:·boolean·`

(prettier/prettier)